jenkins-design-language/src/js/components/CommitId.jsx

```diff
@@ -9,8 +9,14 @@ const GIT_HASH_REGEX = /\b[0-9a-f]{5,40}\b/;
  */
 export class CommitId extends Component {
     render() {
-        const {commitId, url, title} = this.props;
+        const {className, commitId, url, title} = this.props;
         let displayValue;
+        const classNames = ['hash'];
+
+        if (className) {
```

Comment:
Nice, this guard ensures omitting `className` prop won't fail. I think we have unit tests that verify that. 👍